src/dotnet/commands/dotnet-install/PackageToProjectFileAdder.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.DotNet.ExecutablePackageObtainer;
+using Microsoft.Extensions.EnvironmentAbstractions;
+
+namespace Microsoft.DotNet.Cli
+{
+
+    internal class PackageToProjectFileAdder : ICanAddPackageToProjectFile
+    {
+        public void Add(FilePath projectPath, string packageId)
+        {
+            if (packageId == null) throw new ArgumentNullException(nameof(packageId));
```

Comment:
Same comment as above; could this have braces around the single line statement?

Reply:
Fixed